client/src/index.jsx

```diff
@@ -33,7 +34,9 @@ function App() {
           <RelatedProducts />
         </StylesProvider>
         <QA />
-        <Ratings />
+        <RatingProvider>
```

Comment:
Can we move the RatingProvider to also include Overview & Related Products? We'll both need access to this context to get the average rating for the star component.